docs/projects/2024/architecture-transition/rpc.md

```diff
@@ -128,76 +77,136 @@ The move to JSON-RPC opens up the possibility for the front end to consolidate m
-| `/api/db/v0/tables/{tableId}/records/`                      | POST        |           |
-| `/api/db/v0/tables/{tableId}/settings/{settingsId}/`        | PATCH       |           |
-| `/api/db/v0/tables/{tableId}/split_table/`                  | POST        |           |
-| `/api/db/v0/tables/{tableId}/type_suggestions/`             | GET         |           |
-| `/api/db/v0/tables/`                                        | GET         |           |
-| `/api/db/v0/tables/`                                        | POST        |           |
-| `/api/ui/v0/connections/{databaseId}/types/`                | GET         |           |
-| `/api/ui/v0/connections/create_from_known_connection/`      | POST        |           |
-| `/api/ui/v0/connections/create_from_scratch/`               | POST        |           |
-| `/api/ui/v0/connections/create_with_new_user/`              | POST        |           |
-| `/api/ui/v0/database_roles/{roleId}/`                       | DELETE      |           |
-| `/api/ui/v0/database_roles/`                                | POST        |           |
-| `/api/ui/v0/queries/{queryId}/shares/{shareId}/`            | PATCH       |           |
-| `/api/ui/v0/queries/{queryId}/shares/{shareId}/regenerate/` | POST        |           |
-| `/api/ui/v0/queries/{queryId}/shares/`                      | GET         |           |
-| `/api/ui/v0/queries/{queryId}/shares/`                      | POST        |           |
-| `/api/ui/v0/reflect/`                                       | POST        | _(none)_  |
-| `/api/ui/v0/schema_roles/{roleId}/`                         | DELETE      | _(none)_  |
-| `/api/ui/v0/schema_roles/`                                  | POST        | _(none)_  |
-| `/api/ui/v0/tables/{tableId}/records/delete/`               | DELETE      |           |
-| `/api/ui/v0/tables/{tableId}/shares/{shareId}/`             | PATCH       |           |
-| `/api/ui/v0/tables/{tableId}/shares/{shareId}/regenerate/`  | POST        |           |
-| `/api/ui/v0/tables/{tableId}/shares/`                       | GET         |           |
-| `/api/ui/v0/tables/{tableId}/shares/`                       | POST        |           |
-| `/api/ui/v0/users/{userId}/`                                | DELETE      |           |
-| `/api/ui/v0/users/{userId}/`                                | GET         |           |
-| `/api/ui/v0/users/{userId}/`                                | PATCH       |           |
-| `/api/ui/v0/users/{userId}/password_reset/`                 | POST        |           |
-| `/api/ui/v0/users/`                                         | GET         |           |
-| `/api/ui/v0/users/`                                         | POST        |           |
-| `/api/ui/v0/users/password_change/`                         | POST        |           |
+| Endpoint                                                    | HTTP Method | Function                                |
```

Comment:
Overall, I'm supportive of the naming scheme you have here. Thanks for going through the details on it!